tools/testsys/src/aws_resources.rs

```diff
@@ -286,7 +319,30 @@ impl AwsK8s {
                             mode: sonobuoy_mode,
                             kubernetes_version: None,
                             kube_conformance_image: self.kube_conformance_image.clone(),
-                            e2e_repo_config_base64: None,
+                            e2e_repo_config_base64: self.e2e_repo_registry.as_ref().cloned().map(
+                                |e2e_registry| {
+                                    base64::encode(format!(
+                                        r#"buildImageRegistry: {}
+dockerGluster: {}
+dockerLibraryRegistry: {}
+e2eRegistry: {}
+e2eVolumeRegistry: {}
+gcRegistry: {}
+gcEtcdRegistry: {}
+promoterE2eRegistry: {}
+sigStorageRegistry: {}"#,
+                                        e2e_registry,
+                                        e2e_registry,
+                                        e2e_registry,
+                                        e2e_registry,
+                                        e2e_registry,
+                                        e2e_registry,
+                                        e2e_registry,
+                                        e2e_registry,
+                                        e2e_registry
+                                    ))
+                                },
+                            ),
```

Comment:
You could do this instead:

```suggestion
dockerLibraryRegistry: {e2e_registry}
e2eRegistry: {e2e_registry}
e2eVolumeRegistry: {e2e_registry}
gcRegistry: {e2e_registry}
gcEtcdRegistry: {e2e_registry}
promoterE2eRegistry: {e2e_registry}
sigStorageRegistry: {e2e_registry}"#,
                                        e2e_registry,
```